utils/utils.js

```diff
@@ -37,3 +37,20 @@ export function sendTyping(channel) {
 export function equalsIgnoreCase(str1, str2) {
   return str1.toUpperCase() === str2.toUpperCase();
 }
+
+/**
+ * Returns a shuffled COPY of arr.
```

Comment:
For additional info to whoever looks
```suggestion
 * Returns a shuffled COPY of arr
 * using modern Fisher-Yates algorithm.
```